lua/astrocommunity/lsp/coc-nvim/init.lua

```diff
@@ -40,6 +40,7 @@ return {
         maps.n["<Leader>lR"] = maps.n.gr
         maps.n["<Leader>lr"] = { "<Plug>(coc-rename)", desc = "Rename current symbol" }
         maps.n["<Leader>lf"] = { function() vim.cmd.Format() end, desc = "Format buffer", nowait = true }
```

Comment:
```suggestion
        maps.n["<Leader>lf"] = { function() vim.cmd.Format() end, desc = "Format buffer" }
```

hehe this is an accident, maybe fix this too in this PR :')